imagedephi/redact/__init__.py

```diff
@@ -1,3 +1,3 @@
-from .redact import iter_image_files, redact_images, show_redaction_plan
+from .redact import create_redact_dir, iter_image_files, redact_images, show_redaction_plan
 
-__all__ = ["iter_image_files", "redact_images", "show_redaction_plan"]
+__all__ = ["iter_image_files", "redact_images", "show_redaction_plan", "create_redact_dir"]
```

Comment:
`create_redact_dir` is basically an implementation detail of `imagedephi.redact.redact`. It doesn't need to be re-exported back to the `imagedephi.redact` module for general use.

If you need to access it for testing, you can always directly:
```import
from imagedephi.redact.redact import create_redact_dir
```

Reply:
fixed in a0c2bf04704b088fb3ba1cfdeb17ff930b8983bf